controller/src/main/java/io/pravega/controller/server/rpc/auth/PravegaInterceptor.java

```diff
@@ -97,18 +95,25 @@ public static String retrieveDelegationToken(String tokenSigningKey) {
             }
     }
 
+    private String getDelegationToken() {
+        Context context = Context.current();
+        return DELEGATION_CONTEXT_TOKEN.get();
```

Comment:
Did you intend to write `return DELEGATION_CONTEXT_TOKEN.get(context);`

Reply:
The default `get` does the same internally. Dropping the earlier line.